### ISSUE TITLE
Can you release the model/code trained on VisDA?

### ISSUE THREAD

**Issue by johnnylu305** [zhiheLu/STAR_Stochastic_Classifiers_for_UDA#1]
Excuse me, we modify the MCD and want to reproduce your results with VisDA. However, we got the same performance in the MCD paper instead of STAR. We may miss something important. Can you release the model/code trained on VisDA?

**Comment by zhiheLu** [comment]
For VisDA, we modify all three linear layers to the distribution way. I guess this may affect performance.

**Comment by johnnylu305** [comment]
@zhiheLu What is your specific structure for all three linear layers including parameters, dropout, batchnorm, and activation function? In addition, you will keep using two classifiers and set the sigma to zero during testing, am I correct?

**Comment by zhiheLu** [comment]
I sent it to you by email.

**Comment by hzphzp** [comment]
> 

Could you please share the code, thx a lot!
